src/components/Event/EventMenu.svelte

```diff
@@ -1,20 +1,42 @@
 <script>
-    export let event
+    import { fade } from "svelte/transition"
+    import Accordion from "../shared/Accordion.svelte"
+
+    // export let event
     export let setActiveView
+    export let toggleMenu
+    export let menuOpen
 </script>
 
-<ul>
+<div id="EventMenuToggle" on:click={toggleMenu}
+    class="absolute -right-3 -top-0 p-1 bg-white text-emerald-900 border-emerald-900 border cursor-pointer rounded-full h-8 w-8 flex items-center justify-center">
```

Comment:
for detailed or complex styles, tailwind classes can get a bit cluttered pretty quickly